### PR TITLE
Added Custom Scrollbar

### PR DESCRIPTION
**Pull request by Saatvik21**
- Fixes: #21 
- I've added a custom scrollbar of orange colour with hover effect. 
- The color can be changed if needed.

## Screenshot & Video 

For Chrome & Mobile Devices :

https://user-images.githubusercontent.com/61618767/119816948-f96a3f00-bf0a-11eb-85ed-c6a763283ed1.mp4

It works on Microsoft Edge as well

Screenshot_10

### CODE REVIEW

File: src/styles/App.scss [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+    background-image: url("../assets/downarrow.png");
+}
+
+/*Hiding Scrollbar on Mobile Screens*/
+
+@media screen and (max-width: 768px) {
+  ::-webkit-scrollbar {
+    display: none;
+  }
+}
```
Comment (praveenscience) [inline src/styles/App.scss:45]: Good work. Can we please have this indented properly according to prettier rules?

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: src/styles/App.scss (hunk 1 above)
Comment (Saatvik21) [reply src/styles/App.scss]: Done 🙂

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
Thanks for the contribution! 🔥

**Pull request merged**